### DISCUSSION TITLE
v0.9.3-rc published to enable Simple Edge Discovery demo at MWC

### DISCUSSION THREAD

**Discussion by Kevsy** [camaraproject/EdgeCloud#200] [General]
v0.9.3-rc (release candidate) has been published to allow implementation of Simple Edge Discovery at Mobile World Congress: release notes [here](https://github.com/camaraproject/EdgeCloud/releases/tag/v0.9.3-rc)

For anyone not developing for MWC, please wait for release v0.9.4 which incorporates the Identity & Consent Management guidelines for security schemes, and will probably change the resource name for `mec-platforms`.

**Comment by trehman-gsma** [comment]
Hi @Kevsy,

A question about the embedded documentation that is in the [v0.9.3-rc YAML](https://github.com/camaraproject/EdgeCloud/blob/v0.9.3-rc/code/API_definitions/Discovery/simple_edge_discovery.yaml#L68).

The documentation states that a successful response should contain a JSON object containing a `MECPlatforms` array with a single member:
```json
    {
      "MecPlatforms": [
        {
          "edgeCloudProvider": "AWS",
          "edgeResourceName": "eu-west-2-wl1-man-wlz-1"
        }
      ]
    }
```

However the schema in the specification has a slightly difference structure (it does not contain the outer `MECPlatforms`):
```
[
  {
    "edgeCloudProvider": "string",
    "edgeResourceName": "string"
  }
]
```

Can we assume that the documentation is out-of-sync and that the schema is the definitive response structure?

**Comment by Kevsy** [comment]
HI @trehman-gsma  thanks for the catch, that is indeed a mistake.

In fact it's the documentation that's correct (and not the schema definition), but note that `MECPlatforms` will be replaced in v0.9.4 with the name decided by the poll in #197 . 

So in summary: if it's easier to go with the schema version, that seems fine, but just be aware it will change in v0.9.4.

**Comment by Kevsy** [comment]
Now captured as issue in #210
